lib/migration/config/v1/executor/network_interface.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"net"
+
+	"github.com/vmware/vic/pkg/ip"
+)
+
+// NetworkEndpoint describes a network presence in the form a vNIC in sufficient detail that it can be:
+// a. created - the vNIC added to a VM
+// b. identified - the guestOS can determine which interface it corresponds to
+// c. configured - the guestOS can configure the interface correctly
+type NetworkEndpoint struct {
+	// Common.Name - the nic alias requested (only one name and one alias possible in linux)
```

Comment:
This is concerning that you should explain what's the purpose of a specific variable. I would prefer too see something like NICAlias string, rather that Common.Name - which is very unclear.
I understand that it may require significant changes everywhere, but if it is possible, I would prefer to use dedicated variables for it. I also would like to hear other opinions on this, since I may not have full context on this.